src/main/java/org/sonarlint/intellij/messages/AnalysisResultsListener.java

```diff
@@ -26,11 +26,12 @@
 import org.sonarlint.intellij.issue.LiveIssue;
 
 @FunctionalInterface
-public interface AllFilesIssuesListener {
-  Topic<AllFilesIssuesListener> ALL_FILES_ISSUES_TOPIC = Topic.create("Files issues changed", AllFilesIssuesListener.class);
+public interface AnalysisResultsListener {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1214)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this interface extend "java.util.function.Consumer<Map>" and remove the functional method declaration. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1711)